web/modules/custom/joinup_migrate/migrations/mapping_table.yml

```diff
@@ -0,0 +1,98 @@
+id: mapping_table
+label: 'Migration mapping table'
+source:
+  plugin: spreadsheet
+  track_changes: true
+  file: '../resources/migrate/D05.04 Joinup Migration Plan_Content Item Mapping Small_v0.1.xlsx'
+  worksheet: 'Collection'
+  origin: A3
+  header_row: 2
+  columns:
+    - 'Type of content item'
+    - Collection_Name
+    - 'Policy domain'
+    - 'New collection'
+    - Delete
+    - Abstract
+    - Logo
+    - Banner
+    - Owner
+    - 'Administration User'
+    - 'Elibrary Creation'
+    - 'Pre Moderation'
+  row_index_column: row_index
+  keys:
+    row_index:
+      type: integer
+    Nid:
+      type: integer
+process:
+  row_index: row_index
+  nid:
+    plugin: mapping_table_nid
+    source: Nid
+  type:
+    plugin: static_map
+    source: 'Type of content item'
+    map:
+      'Advertisement': advertisement
+      'Analytical model': analytical_model
+      'asset node reference': asset_node_reference
+      'Binary poll': advpoll_binary
+      'Blog': blog
+      'CAMSS Assessment': isa_camss_assessment
+      'Case': case_epractice
+      'Checksum': checksum
+      'Community': community
+      'Contact point': contact_point
+      'Context Help': contexthelp
+      'ContextHelp FAQ': contexthelp_faq
```

Comment:
This is a whole bunch of work that is needed for future migrations. Many of these types are not present in the current data set belonging to this ticket. It is not possible for me to validate the correctness of these at this time, but the data looks OK at first glance. If there are any errors here this will become clear when these content types are migrated in the future.